### ISSUE TITLE
Just Shutdown. Not sure why.

### ISSUE THREAD

**Issue by raguay** [ipfs-shipyard/ipfs-desktop#787]
Please describe what you were doing when this error happened.
I was in the Station program getting ready to process some more information on Chrome when the dialog appeared saying there was an error in the application and it needs to restart.

**Specifications**

- **OS**: darwin
- **IPFS Desktop Version**: 0.6.1
- **Electron Version**: 4.0.0
- **Chrome Version**: 69.0.3497.106

**Error**

```
Error: connect ECONNREFUSED 127.0.0.1:5001
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)
```

**Comment by raguay** [comment]
Not sure. This is what happened:

1) I found out about IPFS on GitHub and installed the command line stuff. I
just followed the default install using ipfs-install on my macOS 10.14.2
system (MacBook Air). Everything seems to work except for the mount command.
2) Then, someone I am following on GitHub starred the IPFS-Desktop program.
I downloaded it and ran it.
3) Then, I saw about the Chrome plugin and installed it.
4) Then, I get the message that the application died and to report it.
Which I did.
5) I've re-run the application and it has been stable for about two hours
(well at least no more crashes. I'm still figuring out how to use it while
waiting on jobs to execute). I've ran the public version of IPFS-Pad, but I
can't get a local copy to work at all.

I'm just playing around with the tech right now, but I was seeing if it was
something I want to integrate with an desktop app I'm making - ScriptPad.

Richard
